Functions/xASL_io_dcm2nii.m

```diff
@@ -312,6 +312,9 @@
                     error('Invalid index: the requested volume (%d) exceeds the actual number of files (%d)',max(parms.Keep),nNifties);
                 end
             end
+            
+            % Fix NIFTI file names
+            [niiEntries, niiNames] = xASL_io_dcm2nii_FixFormat(niiEntries, niiPaths, niiNames);

```

Comment:
@jan-petr 
What does `FixFormat` mean? We need to be specific or before we know it ExploreASL is as unreadable as SPM :)